etc/helm/pachyderm/templates/kube-event-tail/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.kubeEventTail.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kube-event-tail
+    suite: pachyderm
+  name: kube-event-tail
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kube-event-tail
+      suite: pachyderm
+  template:
+    metadata:
+      labels:
+        app: kube-event-tail
+        suite: pachyderm
+    spec:
+      containers:
+        - name: kube-event-tail
+          env:
+            - name: DEBUG_ADDRESS
+              value: 0.0.0.0:8081
+            - name: NAMESPACE
+{{- if eq .Values.kubeEventTail.clusterScope true }}
+              value: ""
+{{ else }}
+              value: {{ .Release.Namespace }}
+{{- end }}
+          image: docker.io/jrockway/kube-event-tail:v0.0.6
```

Comment:
Now that I think about it, we will have to push this to the pachyderm namespace at some point.  (We should open up a ticket to do this work; Jose is working on doing this for etcd and pgbouncer as part of our release, so once he defines a way to do this, then we can do the same thing here.)  

The reason for this is because docker has a pretty low pull rate limit, but we pay them to not rate limit anything in the pachyderm namespace.

Reply:
I opened https://pachyderm.atlassian.net/browse/CORE-810 to capture this